react/future/react-0.13.0.d.ts

```diff
@@ -72,18 +75,26 @@ declare module "react" {
     function createFactory<P>(
         type: ComponentClass<P, any>): Factory<P>;
 
+    //
+    // Unfortunately these definitions are too stringent and do not allow a sufficiently flexible syntax for the Element Type
+    // see example project with react-templates at:  https://github.com/bgrieder/react-templates-typescript-test.git
+    //
+    //function createElement<P>(
+    //    type: string,
+    //    props?: P,
+    //    ...children: ReactNode[]): ReactDOMElement<P>;
+    //function createElement<P>(
+    //    type: ClassicComponentClass<P, any> | string,
+    //    props?: P,
+    //    ...children: ReactNode[]): ReactClassicElement<P>;
+    //function createElement<P>(
+    //    type: ComponentClass<P, any>,
+    //    props?: P,
+    //    ...children: ReactNode[]): ReactElement<P>;
     function createElement<P>(
-        type: string,
+        etype: any,
         props?: P,
         ...children: ReactNode[]): ReactDOMElement<P>;
```

Comment:
What error are you getting? I strongly doubt that we need to weaken the type of `createElement` like this. Did you fix the bug that I mentioned on the main thread? (`initialCount` vs `count`)?